Calibration/IsolatedParticles/src/ChargeIsolation.cc

```diff
@@ -361,4 +361,232 @@ namespace spr {
     return isIsolated;
   }
 
+  double chargeIsolation(const edm::Event& iEvent,
+                         const edm::EventSetup& iSetup,
+                         CaloNavigator<DetId>& theNavigator,
+                         reco::TrackCollection::const_iterator trkItr,
+                         edm::Handle<reco::TrackCollection> trkCollection,
+                         const CaloSubdetectorGeometry* gEB,
+                         const CaloSubdetectorGeometry* gEE,
+                         TrackDetectorAssociator& associator,
+                         TrackAssociatorParameters& parameters_,
+                         int ieta,
+                         int iphi,
+                         const std::string& theTrackQuality,
+                         bool
+#ifdef EDM_ML_DEBUG
```

Comment:
I miss the point for having both `EDM_ML_DEBUG` and `bool debug` in this file

Reply:
I shall remove out in a subsequent PR. At that time I shall get rid of EDM_ML_DEBUG statements

Reply:
I meant removing cost statements

Reply:
For such a simple update, why not profiting of this PR and making everything at once?

Reply:
Taking care of this